src/Nest/Aggregations/Bucket/Histogram/HistogramAggregation.cs

```diff
@@ -26,11 +26,8 @@ public interface IHistogramAggregation : IBucketAggregation
 		[JsonProperty("extended_bounds")]
 		ExtendedBounds<double> ExtendedBounds { get; set; }
 
-		[JsonProperty("pre_offset")]
-		long? PreOffset { get; set; }
-
-		[JsonProperty("post_offset")]
-		long? PostOffset { get; set; }
```

Comment:
For binary compatibility reasons, these can't be removed in a major version i.e. they can be removed in NEST 6.x but not 5.x. These should be marked with `[Obsolete("removed in Elasticsearch 2.0. Will be removed in the next major version of NEST")]` or something to that effect, and the `JsonPropertyAttribute`s removed, so that setting them becomes a no-op.